packages/main/src/Select.ts

```diff
@@ -50,27 +48,23 @@ import {
 	LIST_ITEM_POSITION,
 	SELECT_ROLE_DESCRIPTION,
 } from "./generated/i18n/i18n-defaults.js";
-import Option from "./Option.js";
 import Label from "./Label.js";
 import ResponsivePopover from "./ResponsivePopover.js";
 import Popover from "./Popover.js";
-import StandardListItem from "./StandardListItem.js";
+import CustomListItem from "./CustomListItem.js";
 import Icon from "./Icon.js";
 import Button from "./Button.js";
 
 // Templates
 import SelectTemplate from "./generated/templates/SelectTemplate.lit.js";
-
-// Styles
-import selectCss from "./generated/themes/Select.css.js";
 import ResponsivePopoverCommonCss from "./generated/themes/ResponsivePopoverCommon.css.js";
 import ValueStateMessageCss from "./generated/themes/ValueStateMessage.css.js";
 import SelectPopoverCss from "./generated/themes/SelectPopover.css.js";
+
+// Styles
+import selectCss from "./generated/themes/Select.css.js";
 import type FormSupport from "./features/InputElementsFormSupport.js";
 import type { IFormElement, NativeFormElement } from "./features/InputElementsFormSupport.js";
-import type ListItemBase from "./ListItemBase.js";
-import type SelectMenu from "./SelectMenu.js";
-import type { SelectMenuOptionClick, SelectMenuChange } from "./SelectMenu.js";
 
 /**
  * Interface for components that may be slotted inside `ui5-select` as options
```

Comment:
this interface is no longer implemented by any component and for me it's replaced by OptionBase class so I think that we can replace it.  Basically we know when the class is custom by `isCustom` so we can create a typeguard and to use it in the code where we need to know that we have custom item